### PR TITLE
task was solved

### PR DESCRIPTION
**Pull request by vanyachyzh**

### CODE REVIEW

**Comment by vanyachyzh** [comment]
25 line --- I use trim method because a newline appears inside of li for some reason

**Comment by vanyachyzh** [comment]
- [DEMO LINK](https://vanyachyzh.github.io/js_task_parse_list_DOM/)

**Comment by vanyachyzh** [comment]
- [DEMO LINK](https://vanyachyzh.github.io/js_task_parse_list_DOM/)

**Review by etojeDenys — CHANGES_REQUESTED** [review]
you need to sort in desc 

image

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+function getEmployees(list) {
+  return list.reduce((acc, person) => {
+    acc.push({
+      name: person.innerText,
+      position: person.dataset.position,
+      salary: person.dataset.salary,
+      age: person.dataset.age,
+    });
+
+    return acc;
```
Comment (OlEzhA0) [inline src/scripts/main.js:39]: you can do like
```suggestion
    return [
      ...acc, 
      {
        name: person.innerText,
        position: person.dataset.position,
        salary: person.dataset.salary,
        age: person.dataset.age,
      }    
    ];
```

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey! just added few improvements

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**